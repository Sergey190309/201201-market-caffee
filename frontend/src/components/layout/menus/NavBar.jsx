import React from "react";
import { Link } from "react-router-dom";
import { Menu, Icon, Button } from "semantic-ui-react";

import Language from "../../language/Language";

const NavBar = () => {
  return (

    <Menu>
      <Menu.Item
      name='logo'>
        <Link to='/'>
          <Icon name='utensils' size='large' />
        </Link>
      </Menu.Item>

      <Menu.Item name='item01'>
        NavBarItem01
      </Menu.Item>

      <Menu.Item name='item02'>
        NavBarItem02
      </Menu.Item>

      <Menu.Item name='item03'>
        NavBarItem03
      </Menu.Item>

      <Menu.Menu position='right'>
        <Menu.Item
        name='login'>
          <Link to='/login'><Button>Log-In</Button></Link>
        </Menu.Item>

        <Menu.Item
        name='language'>
          <Link to='#'><Language /></Link>
        </Menu.Item>
      </Menu.Menu>
    </Menu>
  )
};

export default NavBar;
