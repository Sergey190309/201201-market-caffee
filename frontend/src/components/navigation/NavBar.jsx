import React from "react";
import { Link } from "react-router-dom";
import { Menu, Button } from "semantic-ui-react";

import Logo from "../nav_items/Logo";
import Language from "../nav_items/Language";
import NavItem from "../nav_items/NavItem";

const NavBar = () => {
  return (
    <Menu>
      <Menu.Item>
        <Logo color='teal' />
      </Menu.Item>
      <Menu.Item color='orange'>
        Test
      </Menu.Item>
      <NavItem name='priceList' title='Menu' link='/pricelist' />
      <NavItem name='pictures' title='Gallery' link='/pictures' />

      <Menu.Menu position="right">
        <Menu.Item name="login">
          <Link to="/login">
            <Button>Log-In</Button>
          </Link>
        </Menu.Item>

        <Menu.Item name="language" color='yellow'>
          <Link to="#">
            <Language />
          </Link>
        </Menu.Item>
      </Menu.Menu>
    </Menu>
  );
};

export default NavBar;
