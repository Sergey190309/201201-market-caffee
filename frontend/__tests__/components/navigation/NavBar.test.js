import React from 'react'

import ShallowRenderer from "react-test-renderer/shallow";

import NavBar from '../../../src/components/navigation/NavBar'


describe('NavBar component', () => {
  const renderer = new ShallowRenderer();
  const result = renderer.render(<NavBar />)
  test("Snapshot", () => {
    expect(result).toMatchSnapshot()
  });
});