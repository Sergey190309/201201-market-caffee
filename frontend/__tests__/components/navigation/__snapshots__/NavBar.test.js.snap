// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar component Snapshot 1`] = `
<Menu>
  <MenuItem>
    <Logo />
  </MenuItem>
  <MenuItem
    color="orange"
    name="Test"
  />
  <NavItem
    link="/pricelist"
    name="priceList"
    title="Menu"
  />
  <NavItem
    link="/pictures"
    name="pictures"
    title="Gallery"
  />
  <MenuMenu
    position="right"
  >
    <MenuItem
      name="login"
    >
      <Link
        to="/login"
      >
        <Button
          as="button"
        >
          Log-In
        </Button>
      </Link>
    </MenuItem>
    <MenuItem
      name="language"
    >
      <Link
        to="#"
      >
        <Language />
      </Link>
    </MenuItem>
  </MenuMenu>
</Menu>
`;
